test/suite/test_rollback_to_stable15.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-2020 MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+from helper import copy_wiredtiger_home
+import wiredtiger, wttest, unittest
+from wiredtiger import stat
+from wtdataset import SimpleDataSet
+from wtscenario import make_scenarios
+
+def timestamp_str(t):
+    return '%x' % t
+
+# test_rollback_to_stable15.py
+# Test that roll back to stable handles updates present in the
+# update-list for both fixed length and variable length column store.
+class test_rollback_to_stable15(wttest.WiredTigerTestCase):
+    conn_config = 'cache_size=20MB,statistics=(all)'
```

Comment:
How does this work for fixed length column?
```
# Test that roll back to stable handles updates present in the
# update-list for both fixed length and variable length column store.
```
Do we only check rollback to stable works on the update chain? If that is the case, how do you ensure we don't run eviction and everything is still in memory?

Reply:
Yes we only check rollback to stable works on the update chain, since the cache is big enough for all the updates to fit it, wiredtiger won't evict anything and everything will still be in the memory. I can update the comment in the test as well, and maybe make the cache size even bigger.

Reply:
@kommiharibabu Why we need to support rts for fixed-length?

Reply:
Yes, it was already supported before durable history for fixed-length column store. We want to maintain the same support even after durable history.